reference/tugboat-configuration/index.md

```diff
@@ -0,0 +1,194 @@
+# Tugboat Configuration File
+
+Tugboat uses a file at `.tugboat/config.yml` in a git repository to configure
+the previews it builds. Each preview is built from the `config.yml` file in its
+own branch, tag, commit, or pull request. This file is required in order for
+Tugboat to build any previews for a repository.
+
+The only top-level key of the configuration file is `services`. It is a required
+key, and is a map of the services required by a preview.
+
+Every service consists of the service's name as the key, and a map as the value.
+The name must be unique in the `services` list, and must also be a valid host
+name containing only the characters a-z, 0-9, and hyphen (`[a-z0-9-]`).
```

Comment:
Just to confirm, no uppercase letters allowed?

Reply:
I added a link with more info about valid host names